<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRemoveEmptyLinesWidget</class>
 <widget class="QWidget" name="CRemoveEmptyLinesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Empty Lines</string>
  </property>
  <widget class="QCheckBox" name="m_pCheckBoxRemoveOnlyWhiteSpace">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove lines containing only whitespace.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSelAllOutputText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearAllOutputText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnRemEmptyLines">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>REMOVE EMPTY LINES</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSelectedFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSaveAs">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelResult">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
